plugin/core/strip_html.py

```diff
@@ -0,0 +1,29 @@
+from html.parser import HTMLParser
```

Comment:
this file was inspired by https://bobbyhadz.com/blog/python-remove-html-from-string